Chrome compiles an ancient copy of the usb ids database into the binary. Use the system copy for compilation instead.
--- a/services/device/public/cpp/usb/BUILD.gn
+++ b/services/device/public/cpp/usb/BUILD.gn
@@ -2,16 +2,15 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-source_ids = "//third_party/usb_ids/usb.ids"
+source_ids = "/usr/share/hwdata/usb.ids"
 generated_ids = "$target_gen_dir/usb_ids_gen.cc"
 
 action("usb_device_ids") {
   script = "//services/device/public/cpp/usb/tools/usb_ids.py"
-  inputs = [ source_ids ]
   outputs = [ generated_ids ]
   args = [
     "-i",
-    rebase_path(source_ids, root_build_dir),
+    source_ids,
     "-o",
     rebase_path(generated_ids, root_build_dir),
   ]
